### ISSUE TITLE
terminal is not opening !

### ISSUE THREAD

**Issue by mrshoya05** [stackblitz/core#2996]
### **Description of Bug**
Provide a concise description of your bug and your project link (if applicable).

### **Screenshots/Screencast**

**Additional Context/Questions**
Add any other context or questions regarding this bug.

**Comment by SOM-fermonte** [comment]
**Description of Bug**
I am not sure it is a bug, but was unable to find steps for troubleshooting. I can create a new project in the dashboard, for example, angular. The project opens but the web container does not seem to spin up. the terminal is blank and the browser window stays gray and non-functional. I can open a project from a GitHub repo and everything works in that case, although those projects don't have a terminal.

**Steps to Reproduce**
Open stackblitz and log in with GitHub credentials in Edge.
Create a new project. I have tried Node.js (blank project) and angular
Project launches and it is created successfully.
Web container does not seem to spin up, or the terminal functionality appears to be blocked by windows or a browser setting.

**Expected Behavior**
Project should launch, terminal should show a message like 'updating dependencies' and progress and then the browser frame should show server spinning up and serving the project.

**Screenshots/Screencast**
As seen below, the terminal and browser frames are empty and unresponsive.
![image](https://github.com/stackblitz/core/assets/24226837/94f81d65-191d-4526-88fc-5e2d1a6e9558)


**Additional Context/Questions**
Problem is present in Edge, but not in Chrome.
It used to work in Edge about two weeks ago (Late February-early March 2024).
I followed all the guides for getting WebContainers to work in a Chromium based browser, to no avail.
More troubleshooting steps or suggestions are welcome.
This is a corporate computer. I have not tested on my personal PC.
![image](https://github.com/stackblitz/core/assets/24226837/042ce67a-e8d2-4d92-8ae7-7ca6963c2698)

**Comment by HeyGarrison** [comment]
Hey there!

One thing i'm curious about is - do you happen to have an ad blocker or VPN connected here? Also, we recently pushed an update that really helps with memory consumption. Curious to hear if that helped this problem at.

**Comment by SOM-fermonte** [comment]
Thanks for your prompt reply @HeyGarrison 
The changes did improve, but it is still a coin toss for me to get the WebContainer to boot properly in Edge. For example, [this repo](https://stackblitz.com/~/github.com/SOM-fermonte/ng14base) does not boot in Edge
![image](https://github.com/stackblitz/core/assets/24226837/a96465c6-b83f-417b-910b-c2b3bfc2c8ec)
Chrome, on the other hand works well.
The answer to your other question is yes. At the office, or at home via VPN, I am behind enterprise network applicances, including a corporate ad blocker.
P.S. If it helps, this variation of the link above does work in Edge: https://stackblitz.com/github/SOM-fermonte/ng14base, which doesn't show the terminal.

**Comment by HeyGarrison** [comment]
Of course! Yeah if it's your VPN blocking our URLs then we may need to get you talking with our self hosted people. We announced this last year and tbh it's getting a lot of usage these days.

Is there a way you could confirm there is a problem with your vpn?

**Comment by SOM-fermonte** [comment]
@HeyGarrison I apologize I never replied to your comment above.
I do not think it was a VPN or networking issue because the same link worked in Chrome but not in Edge. I believe it was an Edge-specific issue because I tested the same repo today and it works as expected in Edge. I am accessing from the office. I will mark this issue a closed.
Thanks again for your prompt replies!

**Comment by SOM-fermonte** [comment]
I guess I can't close it myself. Just for reference, this is my current version of Edge
![image](https://github.com/stackblitz/core/assets/24226837/3f563d27-ae3b-4d74-a034-0de5f7e1166d)
